### ISSUE TITLE
EPG import

### ISSUE THREAD

**Issue by dpeachey** [ciscoecosystem/terraform-provider-aci#46]
Hi,

When I import an EPG to the Terraform state, I miss all the "relation_fv_rs_*" attributes. Also, I noticed that there is nothing in "depends_on" attribute so I miss the dependencies to application profile and contracts.

Is this expected behaviour when importing EPG?

Thanks,

Dan

**Comment by devarshishah3** [comment]
Yes, for importing dependencies you will need to import them separately. I am working on a fix for importing relations.

**Comment by dpeachey** [comment]
Hi,

Did you make any progress on this?

Thanks,

Dan
